modules/standard/Math.chpl

```diff
@@ -547,14 +545,14 @@ module Math {
     return chpl_expm1(x);
   }
 
-  /* Returns the value of the argument `x` multiplied by the number 2 raised to
-    the argument `n` power. */
+  /* Returns the value of the argument `x` multiplied by 2 raised to the
+     argument `n` power, i.e., x * 2**n. */
```

Comment:
Let's also surround `x * 2**n` in double `

Reply:
Also done